l5kit/l5kit/sampling/agent_sampling.py

```diff
@@ -276,8 +276,10 @@ def generate_agent_sample(
         "world_from_agent": world_from_agent,
         "centroid": agent_centroid_m,
         "yaw": agent_yaw_rad,
-        "speed": np.linalg.norm(future_vels_mps[0]),
         "extent": agent_extent_m,
         "history_extents": history_extents,
         "future_extents": future_extents,
     }
+    if len(history_vels_mps) > 0:
```

Comment:
does `history` in the name make sense given it's only one step? Is it because we want to emphasise that it is computed from past?

Reply:
Correct. "past_speed" probably sounds better, will fix.

Reply:
"recent_speed" sounds even better, IMHO.